test/controllers/update/update.review.beneficial.owner.gov.spec.ts

```diff
@@ -0,0 +1,115 @@
+jest.mock("ioredis");
+jest.mock("../../../src/utils/logger");
+jest.mock('../../../src/utils/application.data');
+jest.mock('../../../src/middleware/authentication.middleware');
+jest.mock('../../../src/middleware/company.authentication.middleware');
+jest.mock('../../../src/middleware/service.availability.middleware');
+jest.mock('../../../src/middleware/navigation/update/has.presenter.middleware');
+
+import * as config from "../../../src/config";
+import { getApplicationData, prepareData } from "../../../src/utils/application.data";
```

Comment:
doesn't look like prepareData is used in your controller so not sure its needed to mock it in these tests

Reply:
Ah I see why its used in imported method setBenficialOwnerData :)